### PR TITLE
Support returning GetNewBlindedBlock as SSZ

### PR DESCRIPTION
**Pull request by ajsutton**
## PR Description
Update the rest api framework to support serialising data as either JSON or binary types. Take advantage of it by allowing get new blinded block to return data as SSZ.

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

**Review by tbenr — APPROVED** [review]
LGTM but another check from @rolfyone might be better

File: infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/RestApiRequest.java [hunk 1]
```diff
@@ -75,31 +74,37 @@ public void respondAsync(final SafeFuture<AsyncApiResponse> futureResponse) {
   public void respondOk(final Object response, final CacheLength cacheLength)
       throws JsonProcessingException {
     context.header(Header.CACHE_CONTROL, cacheLength.getHttpHeaderValue());
-    respond(HttpStatusCodes.SC_OK, JSON_CONTENT_TYPE, response);
+    respond(HttpStatusCodes.SC_OK, response);
```
Comment (rolfyone) [inline infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/RestApiRequest.java]: static import ok?

Comment (ajsutton) [reply infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/RestApiRequest.java]: Do you mean we should use a static import for `SC_OK`?

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**